Gem/Settings/Types/NodeMetrics.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import BigInt
+
+struct NodeMetrics {
```

Comment:
```suggestion
struct NodeStatus {
```

maybe this should a enum?

case (block, latency)
case (error)

Reply:
We can't use NodeStatus, since it's already taken and generated via typeshare, updated to NodeStatusInfo